src/selectors/layout.test.js

```diff
@@ -25,29 +25,12 @@ describe('Selectors', () => {
   });
 
   describe('getSidebarWidth', () => {
-    const { open, closed } = sidebarWidth;
-
-    describe('if sidebar is visible', () => {
-      it(`reduces the chart width by ${open} on screens wider than ${sidebarBreakpoint}`, () => {
-        expect(getSidebarWidth(true, 1200, sidebarWidth)).toEqual(open);
-        expect(getSidebarWidth(true, 900, sidebarWidth)).toEqual(open);
-      });
-
-      it(`sets sidebar width to ${closed} on screens smaller than ${sidebarBreakpoint}`, () => {
-        expect(getSidebarWidth(true, 480, sidebarWidth)).toEqual(closed);
-        expect(getSidebarWidth(true, 320, sidebarWidth)).toEqual(closed);
-      });
+    it(`if visible is true returns the 'open' width`, () => {
+      expect(getSidebarWidth(true, sidebarWidth)).toEqual(sidebarWidth.open);
```

Comment:
I'm not entirely sure if the new test covers the same scenerios as covered by the old tests with the different break points - the tests seems more like a generic unit test?

Reply:
The function changed so the old tests here were no longer relevant, but good catch anyway.